apps/aehyperchains/src/aehc_parent_mng.erl

```diff
@@ -2,100 +2,319 @@
 %%%-------------------------------------------------------------------
 %%% @copyright (C) 2020, Aeternity Anstalt
 %%% @doc
-%%% <h3>Hyperchains parent's process manager</h3>
-%%% See the pattern "ProcessManager" (https://www.enterpriseintegrationpatterns.com/patterns/messaging/ProcessManager.html).
-%%% This component is responsible for orchestration the abstract parent's chain layer through:
-%%%  a) dedicated workers (trackers);
-%%%  b) supplied interface providers (connectors);
-%%% - The first configured view acts as the "master view" and is eligible to dictate the election period through event's announcement;
-%%% - A tracker performs synchronization from the highest known block until genesis pointer through "previous_hash" property;
-%%% - To be able to run the Hyperchains the system must satisfy the connector's acceptance criteria;
-%%%     a) default mode: get_top_block/0, get_block_by_hash/1l
-%%%     b) delegate mode: default mode + dry_send_tx/1;
-%%% - Interested developers can develop their own connectors for a particular parent chain;
-%%% (TODO: To supply link to the official connector's development guide);
+%% Parent chains process manager
+%% This component is responsible for orchestration of the Hyperchains backends (parent chains) through:
+%% - dedicated state machines (trackers);
+%% - blockchain interfaces (https://github.com/aeternity/aeconnector/wiki);
+%% The component traverses parent chain data in lazy evaluation mode (on demand)
+%% TODO: To provide HTTP API for scheduled commitments
+%% TODO: To show dialog about registry record (show notify + address in telegram, throw error on commitment)
+%% a) Setup #1 (monolith)
+%% b) Setup #2 (replica)
+%% c) Setup #3 (shard: history keeper/election master)
+%% Used patterns:
+%% - https://www.enterpriseintegrationpatterns.com/patterns/messaging/ProcessManager.html)
 %%% @end
 %%%-------------------------------------------------------------------
 -module(aehc_parent_mng).
 
--behaviour(gen_server).
+-include("../../aecore/include/blocks.hrl").
```

Comment:
Can't we have `include_lib("apps/aecore/...")`?

Reply:
Second this, include_lib is the way.

Reply:
There is no fetched "dependencies" in the common sense of this term. The whole application is build in [monorepo](https://en.wikipedia.org/wiki/Monorepo) repository structure and deployed as monolith codebase at once (modules are not supposed to rearrange or to be fetched via independent ways). 

This project was reviewed by one of the Erlang language [creators](https://github.com/aeternity/aeternity/commits?author=rvirding) at the beginning, please don’t think that you invent something new with "include_lib" statement. It's a my opinion and I will pursue this way, thanks.